masterlist.yaml

```diff
@@ -301,3 +305,6 @@ plugins:
       - <<: *patchAvailable
         subs: [ 'DarkerNights.esp' ]
         condition: 'active("Radiant Clouds and Fogs.esp") and active("DarkerNights.esp") and not active("DarkerNights-Radiant.esp")'
+  - name: 'Scrap Everything.esp'
+    msg: [ *doNotClean ]
```

Comment:
I'm thinking it's valid not only because he is the author (which I wouldn't have noticed either) but because in Skryim there is a flag you set so that the Cell isn't reset.  That flag has to be set for things like a player home so that items you put in an empty chest don't disappear once you revisit the Cell.  Until the Fallout 4 CK is released this is probably the best the author can do for now.